merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -83,6 +83,22 @@ export class Windows {
       expression: this.__astNode,
     });
   }
+
+  public longerThan(duration: number) : Windows {
```

Comment:
Let's use the Duration type for durations (even though it is currently aliased to `number`)

```suggestion
  public longerThan(duration: Duration) : Windows {
```